frontend/src/services/orderService.js

```
@@ -0,0 +1,21 @@
+    return response.data;
+  } catch (error) {
+    throw error;
+  }
+};
+
+export const getOrderStatus = async (orderId) => {
+  try {
+    const response = await axios.get(`${API_URL}/orders/${orderId}`);
+    return response.data;
+  } catch (error) {
+    throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)